src/seb/cli/table.py

```diff
@@ -0,0 +1,116 @@
+from typing import Optional
+
+import pandas as pd
+from rich.console import Console
+from rich.table import Table
+
+import seb
+from seb.registries import get_all_models
+
+
+def get_main_score(task: seb.TaskResult, langs: Optional[list[str]]) -> float:
+    if langs is None:
+        _langs = task.languages
+    else:
+        _langs = set(langs) & set(task.languages)
+    return task.get_main_score(_langs) * 100
+
+
+def open_source_to_string(open_source: bool) -> str:
+    return "✓" if open_source else "✗"
+
+
+def benchmark_result_to_row(
```

Comment:
are these copies from the docs/run_benchmark? (if so I go to that script and change it such that it imports from here. Note that is perfectly fine add more columns than needed and then removing them at the outermost function.

Reply:
Yes they are, but it's not exactly the same, I would happily keep them separate for now, a little code duplication doesn't hurt, but it would be a shame if we coupled these up and then figured out they have to diverge.

Reply:
Hmm, I think there is a good seperation here. We e.g. converting benchmark results to a row should be a method on the BenchmarkResult object.

Reply:
(though I agree that some duplication can be perfectly valid)

Reply:
Yeah but then it should return a dict, right? Then we can accumulate into a DF like:
```python
records = [res.to_row() for res in benchmark_results]
df = pd.DataFrame.from_records(records)
 ```

Reply:
Hmm right. I would do `to_dict` instead (hmm maybe there is a better name). Then we can also add the a formatter afterward (e.g. for converting open-source from bool to "check")

```python
records = [res.to_dict() for res in benchmark_results]
records = [format(d) for d in records]
df = pd.DataFrame.from_records(records)
```

Reply:
> I think it would generally be a good idea also because we depend a lot in the table code on implementation details of the BenchmarkResult object, which some would say is a big NoNo.

Unsure what you mean by this

Reply:
But generally I still stand by keeping the docs and this a separate code  base. We should only abstract away from the concrete use case if we expect that these two should always change together and we would put more effort into keeping them up to date with each other than reasonable. I would not yet put all my eggs in one basket. It could be that the table in the CLI needs to change in ways different from the online table, and then abstraction will be a pain in the ass for us to deal with.

Reply:
Let us keep them separate for now.